Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use the official Ruby 3.2 image as a base image
+ARG RUBY_VERSION=3.2
+FROM ruby:${RUBY_VERSION}-bullseye
+
+# Set the working directory
+WORKDIR /usr/src/app
+
+# Install system packages
+RUN apt-get update -qq && \
+    apt-get install -y default-mysql-client postgresql postgresql-contrib vim && \
+    apt-get clean
+
+# Copy all files
+COPY . .
+
+RUN bundler config set --local without "db2 oracle sqlserver sqlite postgresql"
```

Comment:
If/when we want this to work out of the box with `postgresql` this would need to be removed.